neonvm/runner-image-loader/daemonset.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: neonvm-runner-image-loader
+  namespace: neonvm-system
+spec:
+  selector:
+    matchLabels:
+      name: neonvm-runner-image-loader
+  template:
+    metadata:
+      labels:
+        name: neonvm-runner-image-loader
+    spec:
+      containers:
+      - image: runner:dev
```

Comment:
Do we have any other images in our system? I know `compute-init` is now replaced by initScript :smiley: 

`crictl` container uses the same image? Also, `runner` is not customizable my cplane, only `rootdisk` image, correct?

Reply:
> Do we have any other images in our system?

In terms of per-compute images, we still have the actual compute image itself. And there's the various images that are only a few per cluster (autoscaler-agent, scheduler, neonvm-controller, neonvm-vxlan-controller, cplane, proxy - and maybe more?)

> `crictl` container uses the same image?

Yup

> Also, `runner` is not customizable by cplane, only `rootdisk` image, correct?

_Technically_ `runner` is customizable (VM spec supports it), but that's not currently hooked up anywhere. `rootdisk` image customization is fully hooked up though.